ToDoList/tests/ToDoList.Test/PutTests.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Mvc;
+using ToDoList.Domain.DTOs;
+using ToDoList.Domain.Models;
+using ToDoList.WebApi.Controllers;
+using Xunit;
+
+namespace ToDoList.Test
+{
+    public class PutTests
+    {
+        [Fact]
+        public void Put_Item_ReturnsItem()
+        {
+            // Arrange
+            var controller = new ToDoItemsController();
+            var toDoItem = new ToDoItem
+            {
+                ToDoItemId = 1,
+                Name = "Pondeli",
+                Description = "Vstavat!",
+                IsCompleted = true
+            };
+            controller.Items.Add(toDoItem);
+            var updateRequest = new ToDoItemUpdateRequestDto("Pondeli", "Day off!", false);
+
+            // Act
+            var result = controller.UpdateById(1, updateRequest);
+            var okResult = result.Result as OkObjectResult;
+            var okResultValue = okResult.Value as ToDoItemUpdateResponseDto;
+
+            // Assert
+            Assert.NotNull(okResult);
+            Assert.IsType<OkObjectResult>(okResult);
```

Comment:
toto neviem či úplne dáva zmysel, pretože predtým si ho pretypovala na OkObjectResult 
var okResult = result.Result as OkObjectResult;

Reply:
Tady je to mysleno tak, ze to safely pretypuju a pak si zkontroluju, jestli se pretypovani podarilo a je to tedy OkObjectResult. Dava to smysl?